plugin/evm/vm.go

```diff
@@ -775,6 +775,7 @@ func (vm *VM) preBatchOnFinalizeAndAssemble(header *types.Header, state *state.S
 		rules := vm.chainConfig.AvalancheRules(header.Number, header.Time)
 		if err := vm.verifyTx(tx, header.ParentHash, header.BaseFee, state, rules); err != nil {
 			// Discard the transaction from the mempool on failed verification.
+			log.Debug("discarding tx from mempool on failed verification", "txID", tx.ID(), "error", err.Error())
```

Comment:
```suggestion
			log.Debug("discarding tx from mempool on failed verification", "txID", tx.ID(), "err", err)
```

same suggestion for the others as well. I see the codebase is actually a bit inconsistent on "err" vs. "error" but we do not need to call `err.Error()` since the logger will stringify for us.

Reply:
thanks